tests/mysql-command/test.rules

```diff
@@ -0,0 +1 @@
+alert mysql any any -> any any (msg:"test mysql";mysql.command; pcre:"/(?:[1-9]\d{5})(?:(?:1[89]\d{2}|2\d{3})(?:0[1-9]|1[012])(?:0[1-9]|[12][0-9]|3[01]))\d{2}(?:\d)(?:[0-9xX])/i""; sid:1;)
```

Comment:
Why use a PCRE and not a fixed pattern ?

Reply:
Because this signature is from my project, and I want to match the China identify card.